SecurityEvents::INTERACTIVE_LOGIN → AuthenticationTokenCreatedEvent (#3243)
ezsystems/developer-documentation: docs/users/user_authentication.md

Depending on your context, you either want to create and return an Ibexa user, or return an existing user, even a generic one.

> Whenever a user is matched, Symfony initiates an `AuthenticationTokenCreatedEvent` event during authentication.

Author: ```suggestion
Whenever a user is matched and authenticated, Symfony initiates an `AuthenticationTokenCreatedEvent`.
```
